src/platform/plugins/shared/presentation_util/public/components/labs/project_list.tsx

```diff
@@ -62,7 +64,39 @@ export const ProjectList = (props: Props) => {
 
   return (
     <EuiFlexGroup direction="column" gutterSize="none" responsive={false}>
-      {items.length > 0 ? <ul>{items}</ul> : <EmptyList solutions={solutions} />}
+      {items.length > 0 ? (
+        <ul>
+          {items}
+          {items}
```

Comment:
We are duplicating the `items` by accident 😄 

```suggestion
          {items}
```

![image](https://github.com/user-attachments/assets/23bf5f92-7e2b-4d4c-a693-8ac685e17a6b)